trashtalk/utils.py

```diff
@@ -18,7 +18,6 @@
 status = StatusCodes(200, 201, 400, 403, 404)
 geolocator = Nominatim()
 google = GoogleV3(api_key=os.getenv('GOOGLE_MAPS_KEY'))
-Point = namedtuple('Point', ['latitude', 'longitude'])
 
```

Comment:
Point is a bit contrived. It deserves to be taken out to simplify the code.

Reply:
Its red because its been removed. It was never used. I think it was left over from some testing I did. `Coordinates` is the only one that's been implemented and used.